src/olympia/blocklist/admin.py

```diff
@@ -419,40 +332,27 @@ def add_view(self, request, **kwargs):
 
         MultiBlockForm = self.get_form(request, change=False, **kwargs)
         is_delete = not self.is_add_change_submission(request, None)
-
         guids_data = self.get_value('guids', request)
         if guids_data and 'input_guids' not in request.POST:
             # If we get a guids param it's a redirect from input_guids_view.
             initial = {key: values for key, values in request.GET.items()}
-            initial.update(
-                **{
-                    'input_guids': guids_data,
-                    'existing_min_version': initial.get('min_version', Block.MIN),
-                    'existing_max_version': initial.get('max_version', Block.MAX),
-                }
-            )
+            initial.update(**{'input_guids': guids_data})
             if 'action' in request.POST:
                 initial['action'] = request.POST['action']
             form = MultiBlockForm(initial=initial)
         elif request.method == 'POST':
+            form_data = request.POST.copy()
```

Comment:
Not in this iteration, no.  (Originally I was trying to get the functionality where you edit and save a Block and it creates a  BlockSubmission working again, but I stripped that out because it was too much extra and a nice to have)